### PR TITLE
CCS-4456 Sign MacOS with new certifacate

### PR DESCRIPTION
**Pull request by msobo1**
Add new certificate wich can be used for Apple Notarisation process

### CODE REVIEW

File: .github/actions/sign/sign.py [hunk 1]
```diff
@@ -34,9 +34,9 @@ def sign_macos(self):
         # Sign libs
         targets = list()
 
-        # Define sign cmd for macOS
-        codesign_cmd = "codesign --force --deep --sign 'Apple Distribution: VOXEET INC. (B55NRA8BRW)' --keychain ~/Library/Keychains/voxeet.keychain-db"
-        codesign_verify_cmd = 'codesign --verify --deep --verbose=4'
+        # Define codesing cmd
```
Comment (kubaau) [inline .github/actions/sign/sign.py:37]: codesign

**Review by kubaau — APPROVED** [review]

File: .github/actions/sign/sign.py [hunk 2]
```diff
@@ -34,9 +34,9 @@ def sign_macos(self):
...
+        codesign_verify_cmd = "codesign --verify --deep --verbose=4"
+        codesign_cmd = "codesign --force --strict --timestamp --sign 'Developer ID Application: VOXEET INC. (B55NRA8BRW)'"
```
Comment (Adam1-3) [inline .github/actions/sign/sign.py:39]: VOXEET? sure?

File: .github/actions/sign/sign.py [hunk 3]
```diff
@@ -50,14 +50,14 @@ def sign_macos(self):
             ret_s = self._sign_target(target, codesign_cmd)
 
             if ret_s != 0:
-                print("ERROR: {} returned {} during signing of {}. Checking another timestamp server.".format(codesign_cmd, ret_s, target))
+                print("ERROR: {} returned {} during signing {}.")
```
Comment (Adam1-3) [inline .github/actions/sign/sign.py:53]: don't we miss `.format(args-for-placeholders)`?

File: .github/actions/sign/sign.py (hunk 2 above)
Comment (msobo1) [reply .github/actions/sign/sign.py:39]: yes, we are sure!

File: .github/workflows/build.yml [hunk 4]
```diff
@@ -46,6 +46,13 @@ jobs:
           -TargetPlatforms=Mac \
           -Package=$(pwd)/${{ env.UNREAL_PLUGIN_OUTPUT }} \
           -StrictIncludes
+      - name: Sign DolbyIO Unreal Plugin
+        env:
+          DOLBYIO_KEYCHAIN_PASSWORD: ${{ secrets.VOXEET_KEYCHAIN_PASSWORD }}
```
Comment (Adam1-3) [inline .github/workflows/build.yml:51]: maybe DOLBY_IO_sth?

Comment (msobo1) [reply .github/workflows/build.yml:51]: It is the name of the keychain, the same as voxeet and others. So, it is DOLBYIO_KEYCHAIN_sth, isn't it?

**Review by kubaau — DISMISSED** [review]

**Review by kubaau — APPROVED** [review]

**Review by Adam1-3 — COMMENTED** [review]
perfect ;)

**Pull request merged**